Sources/MapboxMaps/Gestures/GestureHandlers/PitchGestureHandler.swift

```diff
@@ -1,88 +1,83 @@
-//
-//  PitchGestureHandler.swift
-//  MapboxMapsGestures
-//
-
-import Foundation
 import UIKit
-import CoreLocation
-
-/*
- The `PitchGestureHandler` is responsible for handling the response
- triggered by a gesture comprised of a two-ginger pan in a vertical direction.
- 
- The pitch gesture is only triggered if the angle between the two
- touch points is greater than 45°.
- */
 
-internal class PitchGestureHandler: GestureHandler {
-    internal var initialPitch = CGFloat.zero
-    internal var dragGestureTranslation: CGPoint!
+/// `PitchGestureHandler` updates the map camera in response to a vertical,
+/// 2-touch pan gesture in which the angle between the touch points is less than 45°.
+internal class PitchGestureHandler: GestureHandler, UIGestureRecognizerDelegate {
+    private var initialPitch: CGFloat?
 
-    internal override init(for view: UIView, withDelegate delegate: GestureHandlerDelegate) {
-        super.init(for: view, withDelegate: delegate)
-
-        let pitchGesture = UIPanGestureRecognizer(target: self, action: #selector(handlePitchGesture(_:)))
-        pitchGesture.minimumNumberOfTouches = 2
-        pitchGesture.maximumNumberOfTouches = 2
-        gestureRecognizer = pitchGesture
-        view.addGestureRecognizer(pitchGesture)
+    internal init(gestureRecognizer: UIPanGestureRecognizer,
+                  mapboxMap: MapboxMapProtocol,
+                  cameraAnimationsManager: CameraAnimationsManagerProtocol) {
+        gestureRecognizer.minimumNumberOfTouches = 2
+        gestureRecognizer.maximumNumberOfTouches = 2
+        super.init(
+            gestureRecognizer: gestureRecognizer,
+            mapboxMap: mapboxMap,
+            cameraAnimationsManager: cameraAnimationsManager)
+        gestureRecognizer.delegate = self
+        gestureRecognizer.addTarget(self, action: #selector(handleGesture(_:)))
     }
 
-    @objc internal func handlePitchGesture(_ gesture: UIPanGestureRecognizer) {
-
-        let horizontalTiltTolerance = delegate.horizontalPitchTiltTolerance()
-
-        if gesture.numberOfTouches != 2 {
-            gesture.state = .ended
-            return
+    private func touchAngleIsLessThanMaximum(for gestureRecognizer: UIGestureRecognizer) -> Bool {
```

Comment:
should this be called isTouchAngleIsLessThanMaximum?

Reply:
I don't think so. Typically you look to the place where these are called, so for example, on UIView, there's a property `isHidden`. Grammatically, `is` is the verb and the subject is the object you call the method on (the view) so that it forms a sentence: `view.isHidden`. In the situation you highlighted, we call `touchAngleIsLessThanMaximum` on self, which is a gesture handler. It doesn't really make sense grammatically for there to be a leading `is` on the method name since the function isn't saying anything about the gesture handler itself.